packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -241,8 +144,12 @@ export const createSeatManager = (
     'zcfSeat',
     proposal => ({ proposal }),
     {
-      getNotifier: ({ self }) =>
-        E(zoeInstanceAdmin).getSeatNotifier(zcfSeatToSeatHandle.get(self)),
+      getSubscriber: context => {
+        const { self } = context;
```

Comment:
Since we'll be getting rid of kinds anyway, not a big deal, but:

Since you're not using `context` otherwise, the following would be more stylistically consistent with the other methods below

```suggestion
      getSubscriber: ({ self }) => {
```

Reply:
Done. What does this have to do with `kinds`?

Reply:
Once we switch from kinds to farClasses/farInstances, methods such as this will start out looking more like 

```suggestion
      getSubscriber: () => {
         const { self } = this;
```

which is more like your original rather than my suggested change. Nevertheless, uniformity is the higher immediate priority so I'm glad you did this.